### PR TITLE
Fix broken links on securing-a-cluster page

### PR DESCRIPTION
**Pull request by petermbenjamin**
This PR addresses and fixes some broken links on "Securing a Cluster" page.

### CODE REVIEW

**Comment by petermbenjamin** [comment]
I have completed and signed the CNCF CLA

**Comment by petermbenjamin** [comment]
Went through the preview link provided by the netlify bot: https://deploy-preview-9191--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/securing-a-cluster/

Confirmed that the 2 broken links are now resolving correctly.

File: content/en/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 1]
```diff
@@ -78,20 +78,20 @@ Kubelets expose HTTPS endpoints which grant powerful control over the node and c
 
 Production clusters should enable Kubelet authentication and authorization.
 
-Consult the [Kubelet authentication/authorization reference](/docs/admin/kubelet-authentication-authorization) for more information.
+Consult the [Kubelet authentication/authorization reference](/docs/reference/command-line-tools-reference/kubelet-authentication-authorization) for more information.
```
Comment (liggitt) [inline content/en/docs/tasks/administer-cluster/securing-a-cluster.md]: Hold on this… that link is likely to change

**Comment by liggitt** [comment]
@kubernetes/sig-docs-maintainers how do we set up a redirect from the old kubelet authn location to the current valid page so https://kubernetes.io/docs/admin/kubelet-authentication-authorization is not a dead link?

**Comment by chenopis** [comment]
@liggitt Add a redirect in the `/static/_redirects` file, e.g.:

```
/old/path/to/page/  /path/to/new/page/  301
```

**Comment by petermbenjamin** [comment]
@liggitt - do I have your blessing to revert my changes and add that redirect rule in `/static/_redirects` file instead?

**Comment by petermbenjamin** [comment]
Weird, I could have sworn that link was broken for me, but indeed it seems to redirect properly now.

But, no problem, will add a redirect for the kubelet page.

**Comment by petermbenjamin** [comment]
I will close this PR and open a new one for the redirect.

**Pull request closed without merging**